internal/blockchain/ethereum/config.go

```diff
@@ -56,4 +77,14 @@ func (e *Ethereum) InitPrefix(prefix config.Prefix) {
 	ethconnectConf.AddKnownKey(EthconnectConfigBatchTimeout, defaultBatchTimeout)
 	ethconnectConf.AddKnownKey(EthconnectPrefixShort, defaultPrefixShort)
 	ethconnectConf.AddKnownKey(EthconnectPrefixLong, defaultPrefixLong)
+
+	addressResolverConf := prefix.SubPrefix(AddressResolverConfigKey)
+	wsconfig.InitPrefix(addressResolverConf)
```

Comment:
Do we really need `wsconfig.InitPrefix`, or would `restclient.InitPrefix` suffice? I didn't see any websocket functionality.